### ISSUE TITLE
Menu dropdowns are not closing when clicking on them

### ISSUE THREAD

**Issue by lauramarcos** [chanzuckerberg/cryoet-data-portal#493]
**Steps to Reproduce**
1. Go to Browse all Data page
2. Locate upper menu
3. Click on Tools -> API
4. Dropdown is not closing
5. Click on About & Help -> Go to Documentation

**Expected Results**
Menu dropdowns should close when clicked on them

**Evidence**

https://github.com/chanzuckerberg/cryoet-data-portal/assets/63799148/1c71cd96-7188-47e4-a049-2ba39722633f

**Comment by junxini** [comment]
hey @Janeece, could you remind me if this is expected behavior?

**Comment by Janeece** [comment]
I agree with Laura's recommendation -- menu should close after something is selected.
